wpeverest/everest-forms
# Enhancement - Zoho CRM listing in add-ons page.

### All Submissions:






### Changes proposed in this Pull Request:



Zoho CRM listed on add-ons page.

### How to test the changes in this Pull Request:

Verify, whether Zoho CRM listed or not on the add-ons page. And also verify, whether the installation/activation/deactivation of these add-ons works properly or not.
### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Zoho CRM listing in add-ons page.
